stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -191,8 +191,9 @@ internal struct _ContiguousArrayBuffer<Element> : _ArrayBufferProtocol {
       _storage = Builtin.allocWithTailElems_1(
          _ContiguousArrayStorage<Element>.self,
          realMinimumCapacity._builtinWordValue, Element.self)
-
+      
       let storageAddr = UnsafeMutableRawPointer(Builtin.bridgeToRawPointer(_storage))
+      _swift_stdlib_set_objc_complications_forbidden(storageAddr, 1)
```

Comment:
This will still need an availability check 😔

Reply:
Yeah sadly so :( I wonder if there's anywhere we tail alloc stuff that we DON'T want to do this… if there's not, we could fold it into the builtin maybe? Anyway, availability check isn't the worst thing probably.

Reply:
`ManagedObject` uses the same builtin, and it's probably not a good idea to set the bit on that.

Although... wild idea: what if we set this flag all non-@objc classes?

Reply:
That needs a proposal at the very least.